.gitignore

```diff
@@ -1,3 +1,4 @@
 .Rhistory
 data/
 tad_pathways_data.tar.gz
+figures/
```

Comment:
no, I didn't think it necessary to add the resulting figure from this change into the repo. I could see a figure in the future though, so I will remove the line from the ignore file